website/content/en/docs/tasks/deprovisioning.md

```diff
@@ -4,33 +4,86 @@ linkTitle: "Deprovisioning"
 weight: 10
 ---
 
+Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
+The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
+These include:
 
-## Deletion Workflow
+* Marking the node as unschedulable, so no further pods can be scheduled there.
+* Evicting all pods other than daemonsets from the node.
+* Deleting the node from the Kubernetes cluster.
+* Terminating the instance from the cloud provider.
```

Comment:
Can you move this to before deleting the node?

Reply:
Done.